config/base/role.yaml

```diff
@@ -234,4 +234,22 @@ rules:
   - consoleyamlsamples
   - consoleclidownloads
   verbs:
-  - '*'
\ No newline at end of file
+  - '*'
+- apiGroups:
+  - autoscaling
+  resources:
+  - horizontalpodautoscalers
+  verbs:
+  - '*'
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - '*'
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - '*'
```

Comment:
Is this one needed for base ? (aka for k8s or other non-openshift platform)

Reply:
oops.. no :stuck_out_tongue: 
fixed :+1: